src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1089,6 +1092,13 @@ private void EnforceDoesNotReturn(SyntaxNode? syntaxOpt)
             Analyze(compilation, method, node, diagnostics, useConstructorExitWarnings, initialNullableState, getFinalNullableState, out finalNullableState);
         }
 
+#if DEBUG
+        private static bool ShouldRunAnalysisInDebug(CSharpCompilation compilation)
+        {
+            return compilation.Feature("nullableAnalysis") != "false";
```

Comment:
Consider that in the future this switch is going to have to modes: force on and force off. Don't think `"false"` is a good decriptive value here.